handbook/ops/roles/director_business_operations.md

```diff
@@ -0,0 +1,66 @@
+# Director of Business Operations
+
+Sourcegraph's business operations team empowers our product, design, and engineering organizations with the insights, tools, and data they need to make better decisions. As the Director of Business Operations, you will build the team that helps our product organization scale and serve every developer in the world.
```

Comment:
I think we should significantly broaden the scope of BizOps. It looks to be based off my current focus and our job openings but in reality I see product/eng support as a sub-team within BizOps or Product (where growth teams are sometimes located) and BizOps as a much broader function.

Given that, I would recommend we keep it much more broad so we cater to candidates who can build out teams that are beyond data infrastructure and product support. For example, Notion's BizOps team is:

> The mission of the Business Operations and Finance team is to bring data insight, operational capability and business expertise to promote strong execution. We partner with the executive team to optimize company-wide resource allocation by defining business strategies, driving cross-functional projects, and unlock new growth opportunities. Our team provides strategic perspectives and analytical horsepower to solve critical business problems (such as GTM and pricing strategy) across the organization."

Reply:
Thank you for this! I think we agree on the substance here, so getting a review on my wording is really helpful. Let me think through this wording and I'll propose an update.